src/common/tooltip-utils/tooltip.tsx

```diff
@@ -26,14 +26,10 @@ const Tooltip: FC<TooltipProps> = ({
         throw new Error(`Tooltip: Please use a TooltipHost that defines the host of the tooltip`)
     }
 
-    if (!content) {
-        throw new Error(`Tooltip: Please use a component that defines the content of the tooltip`)
-    }
-
     const host = cloneElement(originalHostComponent, {
         className: `${type}__host`,
         'aria-expanded': isExpanded,
-        'aria-describedby': content.props.id,
+        'aria-describedby': content?.props.id,
```

Comment:
`content?.props?.id`
curious why we dont add check to props aswell

Reply:
We expect that props is always there by default. But let me add it a check there to just in case.